### DISCUSSION TITLE
Welcome to sd-webui-modelscope-text2video Discussions!

### DISCUSSION THREAD

**Discussion by kabachuha** [deforum-art/sd-webui-text2video#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ClashSAN** [comment]
here's another extension working on the same thing, I haven't tested it, but it's good to check out and share ideas - https://github.com/ddPn08/text-to-video-synthesis-webui

**Reply by kabachuha** [reply]
That extension is weirder in the current form, they use an auth token while this extension uses the local models

**Comment by aivrar** [comment]
Hi! I want to try this out but I've got my install in a Docker container using this:
https://github.com/AbdBarho/stable-diffusion-webui-docker

It has a different file tree so the scripts are looking in the wrong place for the models.

**Reply by McMyers183** [reply]
> Hi! I want to try this out but I've got my install in a Docker container using this: https://github.com/AbdBarho/stable-diffusion-webui-docker
> 
> It has a different file tree so the scripts are looking in the wrong place for the models.

I had the same issue but found that It is looking in our "data" directory, so the following edit fixed it for me,

Edit File - Data\config\auto\extensions\sd-webui-modelscope-text2video\scripts\modelscope-text2vid.py 
Change line 30 to - return TextToVideoSynthesis(ph.models_path+' /Stable-diffusion/Modelscope/t2v')

**Reply by aivrar** [reply]
Yes that was it, Thank you. The extension updates often so keep those two files open in an editor to quickly update those too.
